app/lib/features/tasks/widgets/due_chip.dart

```diff
@@ -72,7 +71,7 @@ class _DueChipState extends State<DueChip> {
     } else if (dueDate!.isPast) {
       label = dueDate!.timeago();
       dueTheme = textStyle.copyWith(
-        color: Theme.of(context).colorScheme.taskOverdueFG,
+        color: Theme.of(context).colorScheme.error,
```

Comment:
not sure I agree on that choice. Mid- to long-term we probably want to differentiate between an acter app _error_ and something that should "alert" you within the content. I don't think using `colorScheme.error` for task's being overdue is a good choice ... but I leave that to future decisions about the tasks module.